pytri/__init__.py

```diff
@@ -133,7 +134,66 @@ def clear(self):
         for name in store_layers:
             self.remove_layer(name)
 
-    def axes(self):
+    def _fetch_layer_file(self, fname: str) -> str:
+        """
+        Fetch a layer file from local package resources.
```

Comment:
We assume a specific file structure here, namely that the js file is in pytri/js/. Might be worth mentioning this in the docstring

Reply:
This is a great point!